### ISSUE TITLE
dplyr methods select_ and mutate_ are deprecated

### ISSUE THREAD

**Issue by hlapp** [ropensci/RNeXML#240]
Their use results in the following warnings:
```
mutate_() is deprecated. 
Please use mutate() instead

The 'programming' vignette or the tidyeval book can help you
to program with mutate() : https://tidyeval.tidyverse.org
This warning is displayed once per session.
```
and
```
select_() is deprecated. 
Please use select() instead

The 'programming' vignette or the tidyeval book can help you
to program with select() : https://tidyeval.tidyverse.org
This warning is displayed once per session.
```
@cboettig are you familiar enough with `dplyr` to be comfortable fixing this?

**Comment by cboettig** [comment]
Yup, I can take a look at this.  For the `select_()` commands, changing them to `select()` with no further changes should be all that is necessary.  With `mutate_()` I'll have to take a closer look since it depends on how we're using it...

**Comment by hlapp** [comment]
Reported also in #250, and fixed in #253.
